python/py-sherpa/Portfile

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+PortGroup           compilers 1.0
+name                py-sherpa
+version             1.2
+maintainers         gmail.com:Deil.Christoph openmaintainer
+
+dist_subdir         ${name}/${version}
+
+categories-append   science
+description         Sherpa is a modeling and fitting application for Python
+long_description    Sherpa is a modeling and fitting application for Python. \
+                    It enables the user to construct complex models from \
+                    simple definitions and fit those models to data, using a \
+                    variety of statistics and optimization methods.
+
+platforms           darwin
+license             GPL3+
```

Comment:
Please check with `port lint`:
```
Error: invalid license 'GPL3+': missing hyphen before version
```

The correct license string would be `GPL-3+`.